From 849a657a68d79a998e3e49a8fd079bf1a4b7951c Mon Sep 17 00:00:00 2001
From: Seth Jennings <sjenning@redhat.com>
Date: Mon, 18 Aug 2014 15:42:49 -0400
Subject: [edac] sb_edac: update Kconfig description

Message-id: <1408376571-8244-11-git-send-email-sjenning@redhat.com>
Patchwork-id: 87866
O-Subject: [RHEL7.1 PATCH 10/12] sb_edac: update Kconfig description
Bugzilla: 1127019
RH-Acked-by: Aristeu Rozanski <aris@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>

From: Aristeu Rozanski <aris@redhat.com>

Brew: https://brewweb.devel.redhat.com/taskinfo?taskID=7802951
Bugzilla: https://brewweb.devel.redhat.com/taskinfo?taskID=7802951
Upstream: 66ca72749166163c4ec9e5ab5416548a40d902d3

Kconfig wasn't updated when Ivy Bridge support was added.

Cc: Tony Luck <tony.luck@intel.com>
Signed-off-by: Aristeu Rozanski <aris@redhat.com>
Signed-off-by: Mauro Carvalho Chehab <m.chehab@samsung.com>
Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/edac/Kconfig b/drivers/edac/Kconfig
index a697a64..102d8bc 100644
--- a/drivers/edac/Kconfig
+++ b/drivers/edac/Kconfig
@@ -245,12 +245,12 @@ config EDAC_I7300
    Clarksboro MCH (Intel 7300 chipset).
 
 config EDAC_SBRIDGE
- tristate "Intel Sandy-Bridge Integrated MC"
+ tristate "Intel Sandy-Bridge/Ivy-Bridge Integrated MC"
  depends on EDAC_MM_EDAC && PCI && X86_64 && X86_MCE_INTEL
  depends on PCI_MMCONFIG
  help
    Support for error detection and correction the Intel
-   Sandy Bridge Integrated Memory Controller.
+   Sandy Bridge and Ivy Bridge Integrated Memory Controllers.
 
 config EDAC_MPC85XX
  tristate "Freescale MPC83xx / MPC85xx"
-- 
1.7.1